presentation/src/main/java/me/androidbox/presentation/agenda/screen/AgendaScreenState.kt

```diff
@@ -6,4 +6,5 @@ import java.time.ZonedDateTime
 data class AgendaScreenState(
     val displayMonth: String = ZonedDateTime.now(ZoneId.systemDefault()).month.toString(),
     val usersInitials: String = "",
+    val agendaDropdownItemId: List<Int> = listOf()
```

Comment:
@MKiperszmid @philipplackner Adds the string resources that will be populate the drowdown menu

Reply:
Why do these need to be part of the state? They never change

Reply:
@philipplackner Removed them from the state